lms/djangoapps/ccx/modulestore.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+"""A modulestore wrapper
+
+It will 'unwrap' ccx keys on the way in and re-wrap them on the way out
+
+In practical terms this means that when an object is retrieved from modulestore
+using a CCXLocator or CCXBlockUsageLocator as the key, the equivalent
+CourseLocator or BlockUsageLocator will actually be used. And all objects
+returned from the modulestore will have their keys updated to be the CCX
+version that was passed in.
+"""
+from contextlib import contextmanager
+from ccx_keys.locator import CCXLocator, CCXBlockUsageLocator
+from opaque_keys.edx.locator import CourseLocator, BlockUsageLocator
+
+
+def ccx_locator_to_course_locator(ccx_locator):
+    return CourseLocator(
+        org=ccx_locator.org,
+        course=ccx_locator.course,
+        run=ccx_locator.run,
+        branch=ccx_locator.branch,
+        version_guid=ccx_locator.version_guid,
+    )
```

Comment:
Yes it is.  This was implemented first, and as it became apparent that it would be useful elsewhere I added that API to the key implementation.  I forgot to remove this, but will do so.